data/cncf.yaml

```diff
@@ -2811,7 +2831,8 @@
       url: https://github.com/carvel-dev/carvel
       check_sets:
         - community
-    - name: carvel-kapp
+        - code
```

Comment:
This repository seems to be hosting just some community content and docs. The `code` check set doesn't really work well here. Can you replace `code` with `docs` please?